back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -185,6 +185,61 @@
           end
         end
       end
+
+      context 'with specific publication orderings at different depths' do
+        before do
+          # Purposefully use orderings that don't reflect creation order, to avoid corect ordering due to creation order
+          #
+          # Modelling the structure:
+          # P1 (project)
+          # F1 (folder)
+          # P2 (project)
+          # F2 (folder)
+          #  .. P3-f2 (project)
+          #  .. P4-f2 (project)
+          #  .. P5-f2 (project)
+          # P6 (project)
+          # P7 (project)
+          # P8 (project)
+
+          projects[7].update(title_multiloc: { en: 'P1' })
```

Comment:
We should use `update!` instead of `update`. The problem is that `update` can fail silently, causing issues elsewhere. When this happens, a lot of time can be lost to find back the initial bug. We've had similar cases in the past.